src/util/vscodePrompts.ts

```diff
@@ -11,31 +11,43 @@ export async function askForGroupID(): Promise<string | undefined> {
     value: "com.example",
     ignoreFocusOut: true,
     validateInput: (value: string) => {
-      if (value.trim().length === 0) {
-        return "Group Id is required";
-      }
-      if (value.indexOf(" ") >= 0) {
-        return "Group Id cannot contain a blank space";
-      }
-      return null;
+      return validateGroupId(value);
     },
   });
 }
 
+export async function validateGroupId(groupId: string): Promise<string | undefined> {
```

Comment:
I would move the validateGroupId/validateArtifactId functions to a new `util/validation.ts` file.

Reply:
We should also include a line similar to https://github.com/redhat-developer/vscode-quarkus/blob/master/src/wizards/generateProject/validateInput.ts#L18-L19 in the new file saying where the regex are from

Reply:
i noticed there is already a util.ts, would it be better to add these helpers there instead of creating a new file called validation.ts @rzgry 
thanks

Reply:
Good catch, yeah adding it to utils.ts is also fine. @janamano